samples/bot-type-ahead-search-adaptive-cards/nodejs/Readme.md

```diff
@@ -0,0 +1,78 @@
+---
+page_type: sample
+description: Demonstrating the feature of type ahead search (static and dynamic) control on Adaptive Cards.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+contentType: samples
+createdDate: "24-12-2021 23:30:17"
+---
+
+# Type ahead search control on Adaptive Cards
+
+This sample shows the feature of type ahead search (static and dynamic) control on Adaptive Cards.
+
+`Static search`
+
+![static search card](Images/staticSearchCard.png)
+
+`Dynamic search`
+
+![dynamic search card](Images/dynamicSearchCard.png)
```

Comment:
We have "Submit" button on card so what will happen once we submit it??
Mention in doc for it.